src/Raku/ast/variable-declaration.rakumod

```diff
@@ -751,7 +766,7 @@ class RakuAST::VarDeclaration::Simple
         my @lookups;
         my str $scope := self.scope;
         # If we have a type, we need to resolve that.  Otherwise assume Mu
-        @lookups.push($!type || nqp::null);
+        @lookups.push($!type || RakuAST::Type::Setting.new(RakuAST::Name.from-identifier('Mu')));
```

Comment:
Can just be `@lookups.push($!type || Mu);`
Mu is declared in bootstrap (of which the RakuAST code is part of).

Reply:
If I want to be able to do things like call methods on it (to get it's name, for example), it pays to have it as a RakuAST::Node. But maybe I can just create such a node without passing it to the produce-lookup?

Reply:
I was able to leave 'Mu' out of the implicit lookups without too much trouble.